src/coreclr/tools/Common/JitInterface/CorInfoTypes.cs

```diff
@@ -1151,6 +1151,8 @@ public unsafe struct CORINFO_THREAD_STATIC_BLOCKS_INFO
     {
         public CORINFO_CONST_LOOKUP tlsIndex;
         public uint offsetOfThreadLocalStoragePointer;
+        public nuint tlsGetAddrFtnPtr;
+        public nuint descrAddrOfMaxThreadStaticBlock;
         public CORINFO_CONST_LOOKUP offsetOfMaxThreadStaticBlocks;
```

Comment:
`offsetOfMaxThreadStaticBlocks` is `uint32_t` in the unmanaged definition. This should match.

(pre-existing bug)